services/victoriametrics/scrape_configs.go

```diff
@@ -428,12 +428,34 @@ func scrapeConfigsForPostgresExporter(s *models.MetricsResolutions, params *scra
 		return nil, err
 	}
 
-	// MR and LR is not supported by postgres_exporter, so skipping them.
+	mrOptions := []string{
+		"custom_query.mr",
+	}
+	mrOptions = collectors.FilterOutCollectors("", mrOptions, params.agent.DisabledCollectors)
```

Comment:
I believe there is benefit in extracting out this common functionality in a separate function

Reply:
what do you mean by common functionality? at max we can get rid of a few lines